### PR TITLE
Detect modules

### PR DESCRIPTION
**Pull request by brayo-pip**
Fixes https://github.com/ActivityWatch/aw-tauri/issues/12

---

| :rocket: This description was created by [Ellipsis](https://www.ellipsis.dev) for commit 28a093e63aa488118cb26526bf96c794d391e573  | 
|--------|

### Summary:
This PR enhances module detection by adding functionality to identify and manage 'aw' prefixed executables in the system PATH, and consolidates `tauri` imports for cleaner code.

**Key points**:
- Consolidated `tauri` imports in `src-tauri/src/main.rs`.
- Added `get_modules_in_path` function in `src-tauri/src/manager.rs` to detect 'aw' prefixed executables in PATH.
- Populated `modules_in_path` in `ManagerState` to track and manage executable modules found in system PATH.
- Updated system tray menu creation to include options for starting modules found in PATH but not currently running.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

**Comment by brayo-pip** [comment]
Link the related issue too, I can't. I don't have the necessary permissions I guess

**Comment by ErikBjare** [comment]
@brayo-pip Just mention it in the issue description, like so: "Closes #issue_number"

**Comment by brayo-pip** [comment]
I don't have the gear icons here 
image

**Comment by ErikBjare** [comment]
@brayo-pip I meant like this:

image

File: src-tauri/src/manager.rs [hunk 1]
```diff
@@ -139,8 +144,12 @@ impl ManagerState {
...
 fn send_sigterm(pid: u32) -> Result<(), nix::Error> {
     let pid = Pid::from_raw(pid as i32);
-    signal::kill(pid, Signal::SIGTERM).unwrap();
-    Ok(())
+    let res = signal::kill(pid, Signal::SIGTERM);
+    if let Err(e) = res {
+        return Err(e);
+    } else {
+        return Ok(());
+    }
```
[the remark this answers is not in the document]
Comment (ErikBjare) [reply src-tauri/src/manager.rs:159]: Nice!

File: src-tauri/src/manager.rs [hunk 2]
```diff
@@ -232,3 +239,52 @@ fn start_module_thread(name: String, tx: Sender<ModuleMessage>) {
...
+                        }
+                    }
+                }
+            }
+        }
+    }
+    set.remove("awk"); // common in most unix systems
+
+    set
+}
```
[the remark this answers is not in the document]
Comment (ErikBjare) [reply src-tauri/src/manager.rs:277]: Very good!

**Comment by ErikBjare** [comment]
Very nice! Since you structured your commits and the messages so well, I'll go ahead and skip squash and rebase-merge it :)

**Pull request merged**